lib/pavilion/plugins/commands/show.py

```diff
@@ -390,69 +415,126 @@ def _functions_cmd(self, _, args):
                 title="Available Expression Functions"
             )
 
-    @show_cmd('host')
-    def _hosts_cmd(self, pav_cfg, args):
-        """List all known host files."""
+    def show_vars(self, pav_cfg, config, conf_type):
+
+        file = resolver.TestConfigResolver(pav_cfg)._find_config(conf_type,
+                                                                 config)
+        with file.open() as config_file:
+            config_data = file_format.TestConfigLoader().load(config_file)
+
+        data = []
+        col_names = ['Variables', ' ', 'Type', 'Value']
+
+        name = config
+        path = file
+        config = config_data
+
+        for var in config['variables'].keys():
+            var_out = var
+            index = 0
+            for subvar in config['variables'][var]:
```

Comment:
You spend so much space tracking the index; Just loop over it.
```python
for idx in range(len(config['variables'][var])):
  subvar = config['variables'][var][idx]
```

Then you just have it. No need to increment it at the right places, and you can and should start doing things based on it. For example:
```python
                        data.append({
                            'variables': var_out if idx == 0 else '',
                            ' idx': idx,
                            'Value': subvar,
                        })
```